docs/metrics/prometheus/grafana/minio-dashboard.json

```diff
@@ -93,8 +42,10 @@
   "liveNow": false,
   "panels": [
     {
-      "cacheTimeout": null,
-      "datasource": "${DS_PROMETHEUS}",
+      "datasource": {
+        "type": "prometheus",
+        "uid": "n5pUBg3nz"
+      },
       "description": "",
```

Comment:
You need to make sure to generate the JSON from the grafana dashboard as "generic" export.